### PR TITLE
CASH-176: new command for deleting archived entries; updated exising …

### PR DESCRIPTION
**Pull request by nicuzs**
…delete cmd

### CODE REVIEW

**Comment by nicuzs** [comment]
@cristina0botez  @tobinmori @mvmocanu @horatium

File: src/atris/management/commands/delete_old_historical_records.py [hunk 1]
```diff
@@ -42,7 +50,15 @@ def handle(self, *args, **options):
...
             ))
             return
-        old_history_entries = HistoricalRecord.objects.older_than(days, weeks)
-        handled_entries_nr = old_history_entries.count()
-        old_history_entries.delete()
-        self.stdout.write('{} deleted.\n'.format(handled_entries_nr))
+        from_archive = options.get('from_archive')
+
+        try:
+            model_to_delete = ArchivedHistoricalRecord if \
```
Comment (foobarna) [inline src/atris/management/commands/delete_old_historical_records.py]: why not naming the variable `model_to_delete` to `model`? You are in the delete command, you know what it's usage is.
Why not use `.. = ... if from_archive else ...` - getting rid of `is True` - redundant.
Why keep this in try/except block? It's not necessary.

These 3 changes will make the line fit in 80 chars limit, thus getting rid of `\` annoyance :)

File: tests/tests/functional/commands/test_history_deleter.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def test_delete_older_than_weeks():
                             weeks=1, stdout=out)
     # assert
     assert Poll.history.count() == 1
-    assert '1 deleted.' in out.getvalue()
+    assert '1 HistoricalRecord deleted.\n' in out.getvalue()
```
Comment (cristina0botez) [inline tests/tests/functional/commands/test_history_deleter.py:48]: Please also add a test for the `ArchivedHistoricalRecord` scenario

Comment (nicuzs) [reply tests/tests/functional/commands/test_history_deleter.py:48]: done

**Review by mvmocanu — APPROVED** [review]

**Pull request merged**